.mise/tasks/release

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+# mise description="Releases a particular version of Tuist"
+
+require "pathname"
+require "semantic"
+require "optparse"
+
+options = {}
```

Comment:
I decided to write this one in Ruby for 2 reasons:
- This repo is barely contributed to, so it's ok if we introduce an additional programming language for our convenience.
- We are using Mise, which eases setting up the Ruby environment for anyone contributing here.

Let me know if you'd like me to re-write this one in bash.